draft-ietf-webpush-protocol.xml

```diff
@@ -336,10 +357,17 @@ Link: </p/JzLQ3raZJfFBR0aqvOMsLrt54w4rJUsV>;
         rel="urn:ietf:params:push"
 Link: </receipts/xjTG79I3VuptNWS0DsFu4ihT97aE6UQJ>;
         rel="urn:ietf:params:push:receipt"
+Link: </set/4UXwi2Rd7jGS7gp5cuutF8ZldnEuvbOy>;
+        rel="urn:ietf:params:push:set"
 Location: https://push.example.net/s/LBhhw0OohO-Wl4Oi971UGsB7sdQGUibx
 
 ]]></artwork>
       </figure>
+      <section anchor="subscription_sets" title="Subscription Sets">
+        <t>
+          TODO: Discuss correlation either based on authentication or connection

```

Comment:
Right now, it would be limited to correlation. I probably need another section for "motivation".